test/spec/modules/rubiconAnalyticsAdapter_spec.js

```diff
@@ -224,7 +224,6 @@ const MOCK = {
         'transactionId': 'c116413c-9e3f-401a-bee1-d56aec29a1d4',
         'sizes': [[1000, 300], [970, 250], [728, 90]],
         'bidId': '3bd4ebb1c900e2',
-        'seatBidId': 'aaaa-bbbb-cccc-dddd',
```

Comment:
This is the mock of the `bidRequestObject`, which would never have `seatBidId` in it since `seatBidId` is a bidResponse field.

So not realistic to have it here.